lambda_handlers/docker_authorizer.py

```diff
@@ -0,0 +1,84 @@
+'''
+    Fetch docker token from registry for hub pull and push operation for docker hub
+    Token can be alternately obtained by curl:
+    curl "https://auth.docker.io/token?service=registry.docker.io&scope=repository:jina-ai/jina:pull,push"
+'''
+import re
+import logging
+import urllib3
+import requests
+
+
+def get_logger(context='generic', file=True):
+    logger = logging.getLogger(context)
+    logger.setLevel(logging.DEBUG)
+    return logger
+
+
+def fetch_docker_token():
+    logger = get_logger(context='fetch_docker_token')
+    http = urllib3.PoolManager()
+  
+    headers = {
+        'service': DockerRegistry.SERVICE,
+        'scope': DockerRegistry.SCOPE
+    }
+    response = requests.get('https://auth.docker.io/token?service=registry.docker.io&scope=repository:jina-ai/jina:pull,push')
```

Comment:
Import `requests` would fail here, as deployment package is not built. Please use `http.request` like [here](https://github.com/jina-ai/cloud-ops/blob/master/lambda_handlers/hubapi_authorizer.py#L23)

Reply:
done